postprocessing/janus-pp-rec.c

```diff
@@ -727,13 +727,31 @@ int main(int argc, char *argv[])
 			JANUS_LOG(LOG_VERB, "  -- -- Skipping CSRC list\n");
 			skip += rtp->csrccount*4;
 		}
+		if (rtp->csrccount || rtp->extension) {
+			rtp_read_n = (rtp->csrccount + rtp->extension)*4;
+			bytes = fread(prebuffer+rtp_header_len, sizeof(char), rtp_read_n, file);
+			if (bytes < rtp_read_n) {
+				JANUS_LOG(LOG_WARN, "Missing RTP packet header data (%d instead %"SCNu16")\n",
+					rtp_header_len+bytes, rtp_header_len+rtp_read_n);
+				break;
+			} else
```

Comment:
Code style: the other condition of the `else` also should have a bracket, even of the action is simple (makes the code more readable and less ambiguous).